### DISCUSSION TITLE
Android Auto. Add an option to remove the selector

### DISCUSSION THREAD

**Discussion by CarlosBrys** [organicmaps/organicmaps#7166] [Ideas]
Add an option to remove that selection screen.
I just want to see the map while driving.
I don't want to look for anything 
I don't want to see points of interest.
![IMG_20240111_115944](https://github.com/organicmaps/organicmaps/assets/6495550/5625dd49-6c30-4a3c-b846-2a5c224d2c6b)

**Comment by biodranik** [comment]
@AndrewShkrob on iOS, the default mode is the free driving. Is it possible to implement the same on iOS?

**Comment by AndrewShkrob** [comment]
Honestly, I don't understand the purpose of this free drive mode.
But we can consider creating a separate screen without menu.
There is an ongoing discussion regarding this option: #7064.
Any thoughts and comments are welcome there.

**Reply by biodranik** [reply]
This is the main issue with many software products when developers do not understand their users :) Luckily, Organic Maps is different. Many people use free drive mode, me included. A most popular case is when you're looking for free parking (it's a big issue in some cities), or exploring the environment.

**Reply by CarlosBrys** [reply]
Look:
When your competitors listen to their customers ...

![IMG_20240115_193741](https://github.com/organicmaps/organicmaps/assets/6495550/7a9108cd-6e22-4324-9856-2939f9052c89)

**Comment by muralito** [comment]
> Many people use free drive mode, me included
Me too. I almost never use the "go to" feature and I miss the free drive 3D view of my old Nuvi...
The 2D view is good for pedestrians, but for cars imo it's more useful a 3D view as you can see a larger area in the direction you are heading to.

**Comment by AndrewShkrob** [comment]
Done: https://github.com/organicmaps/organicmaps/pull/7569
